assets/js/src/application.js

```diff
@@ -116,6 +116,8 @@
         transformData: function (hits) {
           return hits.map(function (hit) {
             hit.url = hit.url.replace('https://v4-alpha.getbootstrap.com', '/docs/4.0')
+            var baseurl = $('#search-input').data('baseurl')
```

Comment:
Can you try moving this above and chain the two calls?

Also, can you use vanilla JS? Like getElementById and getAttribute? Maybe like `document.getElementById('search-input').getAttribute(data-baseyrl)` (untested).